nameko/utils.py

```diff
@@ -1,10 +1,11 @@
-import sys
 import inspect
 import re
+import sys
+from pydoc import locate
```

Comment:
interesting. seems to have nice behaviour around missing target vs target having bad imports. could maybe use this in cli.run (separate pr)